smart-chef-ui/src/components/toggle/Toggle.css

```diff
@@ -0,0 +1,57 @@
+.toggle-switch {
+    
+    position: relative;
+    display: inline-block;
+    width: 50px;
+    height: 25px;
+  }
+  .toggle-switch input[type="checkbox"] {
+    display: none;
+  }
+  .toggle-switch .switch {
+    position: absolute;
```

Comment:
Warum ist der toggle-switch bereits vor positioniert.
Einfach den ganzen position Teil rausnehmen uns später in der Seite Relativ zu anderen Elementen definieren.
Hierfür werden dann die Props interessant.